cfgov/jinja2/v1/_includes/atoms/checkbox-list.html

```diff
@@ -0,0 +1,52 @@
+{# ==========================================================================
+
+   common_form_fields.checkbox_list()
+
+   ==========================================================================
+
+   Description: Returns a date picker.
+
+   legend:      Name of the field. Defaults to 'Enter Date Range'.
+
+   required:    Whether the field is required. Defaults to false.
+
+   disabled:    Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% from 'atoms/input.html' import input %}
+{% set options = {
+            'label':       'This is a label',
+            'type':        'checkbox',
+            'helperText':  lipsumText,
+            'placeholder': 'mm/dd/yyy',
+            'size':        '1-2',
+            'required':    true,
+            'disabled':    false,
+            'choices':     ['1','2',3,4],
+            'classes':     'required-check-group'
+} %}
+{% do options.update(value) if value %}
+{% set value = options %}
+{% set id = get_unique_id('_') %}
+<div class="form-l_col {{ 'form-l_col-' ~ value.size
+                          if value.size else 'form-l_col-1' }}">
+    <fieldset class="u-reset {{ value.classes }}">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/65fd027c19ba7c5096d15a32912caee61807f7c4